src/main/context-handler/context-handler.ts

```diff
@@ -132,10 +135,21 @@ export class ContextHandler {
   }
 
   protected async newApiTarget(timeout: number): Promise<httpProxy.ServerOptions> {
+    await this.ensureServer();
```

Comment:
Explicit side-effect `ensureServer` **must** be a dependency instead.

Reply:
Might be out-of-scope for this PR (not sure yet).